velox/type/Variant.cpp

```diff
@@ -187,6 +187,11 @@ std::string variant::toJson(const TypePtr& type) const {
     return '"' + DATE()->toString(value<TypeKind::INTEGER>()) + '"';
   }
 
+  VELOX_USER_CHECK_EQ(
+      this->kind(),
+      type->kind(),
+      "variant TypeKind does not match input TypeKind");
```

Comment:
nit: perhaps, "Wrong type in variant::toJson"

Reply:
Fixed.